src/main/java/org/sagebionetworks/web/client/PopupUtilsView.java

```diff
@@ -10,6 +10,8 @@
 public interface PopupUtilsView {
 	void notify(String message, DisplayUtils.NotificationVariant variant, ToastMessageOptions options);
```

Comment:
Would love to get rid of some of these signatures, but if there are many references that have varying requirements then I agree that it should be another project.